### PR TITLE
BAH-2150 | Updated fhir2 version to 1.5.0

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: omod/pom.xml [hunk 1]
```diff
@@ -67,7 +67,7 @@
         <dependency>
             <groupId>org.openmrs.module</groupId>
             <artifactId>fhir2-omod</artifactId>
-            <version>1.4.0-SNAPSHOT</version>
+            <version>1.5.0-SNAPSHOT</version>
```
Comment (binduak) [inline omod/pom.xml:70]: @SanoferSameera 1.5.0 is released. You can change from SNAPSHOT to release version

File: omod/src/main/resources/config.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 	<require_modules>
 		<require_module>org.openmrs.module.webservices.rest</require_module>
-		<require_module version="1.4.0-SNAPSHOT">org.openmrs.module.fhir2</require_module>
+		<require_module version="1.5.0-SNAPSHOT">org.openmrs.module.fhir2</require_module>
```
Comment (binduak) [inline omod/src/main/resources/config.xml:25]: same as above

File: pom.xml [hunk 3]
```diff
@@ -64,13 +64,13 @@
 			<dependency>
 				<groupId>org.openmrs.module</groupId>
 				<artifactId>fhir2-api-2.1</artifactId>
-				<version>1.4.0-SNAPSHOT</version>
+				<version>1.5.0-SNAPSHOT</version>
```
Comment (binduak) [inline pom.xml:67]: same as above

File: pom.xml [hunk 4]
```diff
@@ -64,13 +64,13 @@
...
 				<scope>provided</scope>
 			</dependency>
 			<dependency>
 				<groupId>org.openmrs.module</groupId>
 				<artifactId>fhir2-api</artifactId>
-				<version>1.4.0-SNAPSHOT</version>
+				<version>1.5.0-SNAPSHOT</version>
```
Comment (binduak) [inline pom.xml:73]: same as above

File: omod/pom.xml (hunk 1 above)
Comment (SanoferSameera) [reply omod/pom.xml]: done

**Pull request merged**